docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc

```diff
@@ -1,18 +1,40 @@
 
 = BGP Monitoring Protocol
+:description: Learn how the BGP Monitoring Protocol (BMP) integration in OpenNMS Horizon/Meridian helps you to see what the Border Gateway Protocol is doing.
 
-The BGP Monitoring Protocol (BMP) provides a convenient interface for obtaining route views.
-The integration in {page-component-title} lets you use these route views, status updates, and statistics for monitoring and management.
+The https://datatracker.ietf.org/doc/html/rfc7854[BGP Monitoring Protocol] (BMP) monitors Border Gateway Protocol (BGP) sessions and provides a convenient interface to monitor BGP routing information on the routing device.
+
+The BMP integration in {page-component-title} lets you use these route views, status updates, and statistics for advanced monitoring and management, providing visibility into what BGP is doing.
+
+== {page-component-title} BMP integration
+
+{page-component-title} leverages https://www.snas.io/[OpenBMP], an application that was developed as a reference implementation for receiving BMP messages from network equipment and presenting the state of the BGP network to the user.
+
+BGP runs on routers and high-end switches.
+Those devices can send BMP data to {page-component-title} directly or through a {page-component-title} Minion collector.
 
 .BMP integration overview
-image::bmp/bmp_integration_overview.png[width=70%]
+image::bmp/bmp_integration_overview_2.png[width=70%]
+
+The {page-component-title} BMP parser accepts BMP connections from router packets using a TCP listener.
+Three adapters in OpenNMS read BMP data, including an inventory list of connected routers and the peers they are established with, and stats and data about what is being exchanged over those peers:
+
+* *BMP Telemetry Adapter:* Handles BMP statistics the BMP parser has received and parsed.
+Statistics received from the router are associated as performance data with that router.
+View metrics and statistics related to that peer and apply thresholding.
+
+* *BMP Peer Status Adapter:* Handles BMP Peer Up and Down messages that the BMP parser receives and parses, and converts them to OpenNMS events.
+Allows alarm creation based on up/down status.
+
+//I notice that Integration Adapter is missing from the diagram. Should we add it? Has the architecture changed?
+
+* *OpenBMP Integration Adapter:* Integrates with an existing OpenBMP installation and handles BMP messages from the BMP parser, and creates OpenBMP-compatible messages, which are then passed to the OpenBMP Kafka cluster.
 
-The following sections provide details about associated parsers, adapters, and listeners.
+The parser and adapters are configurable in the telemetryd-configuration.xml file.
 
-== Distributed BMP processing with Sentinel
+== Before you begin
+To use {page-component-title} BMP features, you need the OpenBMP https://github.com/SNAS/obmp-postgres[PostgreSQL component] installed and set up, and have a BMP-enabled router.
```

Comment:
we don't need OpenBMP postgres component. I think this is only needed for OpenBMP Integration Adapter.